spec/RateLimit.spec.js

```diff
@@ -0,0 +1,283 @@
+describe('rate limit', () => {
+  it('can limit cloud functions', async () => {
+    Parse.Cloud.define('test', () => 'Abc');
+    await reconfigureServer({
+      rateLimitOptions: [
+        {
+          path: '/functions/*',
+          windowMs: 10000,
+          max: 1,
+          message: 'Too many requests. Please try again later',
+        },
+      ],
+    });
+    const response1 = await Parse.Cloud.run('test');
+    expect(response1).toBe('Abc');
+    await expectAsync(Parse.Cloud.run('test')).toBeRejectedWith(
+      new Parse.Error(Parse.Error.CONNECTION_FAILED, 'Too many requests. Please try again later')
+    );
+  });
+
+  it('can add global limit', async () => {
+    Parse.Cloud.define('test', () => 'Abc');
+    await reconfigureServer({
+      rateLimitOptions: {
+        windowMs: 10000,
+        max: 1,
+        message: 'Too many requests. Please try again later',
```

Comment:
Where does the `Please try again later` part come from? Is that a default message by the rate limiting library? I would shorten this to just `Too many requests.`.